boards/arm/reel_board/dts/reel_board.dtsi

```diff
@@ -8,25 +8,38 @@
 / {
 	leds {
 		compatible = "gpio-leds";
-		red_led: led_0 {
+		back_led: led_0 {
```

Comment:
thanks @henrikbrixandersen, please do not add a node for the back led, I removed this in f467f07e1444c63c4937fe788f55bdba9ab6f405 because on the new board version 1507.3 this pin controls a buzzer.

Reply:
@jfischer-phytec-iot Is board version 1507.3 the same as `reel_board_v2` in Zephyr?

Reply:
@henrikbrixandersen reel_board_v2 is 1507.2 or 1507.3 with GDEH0213B72 display. The deference between 1507.2 and 1507.3 is the buzzer.

Reply:
@jfischer-phytec-iot Thanks. I have updated the suggested changes accordingly.